recipes/upsetplot/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "UpSetPlot" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ad40e6c5bc614bb38dff10d1c4a12ab3e20ea2f978ada8d67ba5ad3bbf195f2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pandas
+    - matplotlib >=2.0
```

Comment:
Do you really need matplotlib and pandas to be able to build?

Reply:
Probably not!